api/reviews/stentor.api.md

```diff
@@ -0,0 +1,246 @@
+## API Report File for "stentor"

```

Comment:
Is this supposed to be a `.md` file?

Reply:
I think so. This is a generated file from the api-extractor library, see [here](https://api-extractor.com/pages/overview/demo_api_report/).  We aren't using them yet but what they will do is expose any public facing API changes in PRs.  We will add this to the build process so we can review this file when reviewing PRs to make sure we aren't removing features of the API and breaking things for consumers of the API.